ci-operator/config/openshift/driver-toolkit/openshift-driver-toolkit-master.yaml

```diff
@@ -44,6 +44,11 @@ tests:
   commands: make verify
   container:
     from: src
+- as: check-commits-count
+   commands: /ci/prow/check-commits-count
+   container:
+     from: docker.io/alpine/git:latest 
```

Comment:
Seems like this is not working https://docs.ci.openshift.org/docs/how-tos/external-images/.
A simple work around will be to just `dnf install git` before running `ci/prow/check-commit-count`. WDYT?